src/test/base58_tests.cpp

```diff
@@ -81,19 +86,26 @@ BOOST_AUTO_TEST_CASE(base58_DecodeBase58)
     BOOST_CHECK(!DecodeBase58Check("3vQB7B6MrGQZaxCuFg4oh\0" "0IOl"s, result, 100));
 }
 
-BOOST_AUTO_TEST_CASE(base58_random_encode_decode)
+BOOST_AUTO_TEST_CASE(base58_random_encode_decode_with_optional_spaces)
```

Comment:
I don't think this change makes sense, since there is already other stuff added to the test data. I would just leave it as is.